docs/reference/commandline/volume_inspect.md

```diff
@@ -12,7 +12,7 @@ Usage:  docker volume inspect [OPTIONS] VOLUME [VOLUME...]
 Display detailed information on one or more volumes
 
 Options:
-  -f, --format string   Format the output using the given Go template
+  -f, --format string   Format the output using the given Go template (default "json")
```

Comment:
Also missed this one?

Reply:
The inspect commands are different, they can't show the output in a table. I updated the inspect commands help to output:
```console
❯ ./build/docker volume inspect --help 

Usage:  docker volume inspect [OPTIONS] VOLUME [VOLUME...]

Display detailed information on one or more volumes

Options:
  -f, --format string   Format output using a custom template:
                        'json':             Print in JSON format
                        '<TEMPLATE>':       Print output using the given Go template.
                        Refer to https://docs.docker.com/go/formatting/ for more information about formatting output with templates (default "json")
```